engine/src/gldrv/gl_init.cpp

```diff
@@ -271,39 +283,45 @@ void init_opengl_extensions() {
     } else {
 #ifdef __APPLE__
 #ifndef __APPLE_PANTHER_GCC33_CLI__
-        glLockArraysEXT_p   = 0;
-        glUnlockArraysEXT_p = 0;
+        glLockArraysEXT_p   = nullptr;
+        glUnlockArraysEXT_p = nullptr;
+#else
+        glLockArraysEXT_p = nullptr;
+        glUnlockArraysEXT_p = nullptr;
```

Comment:
do we really need the preprocessor block here? Or can we do away with it and just initialize them to nullptr?

Reply:
no, as a matter of fact, I don't believe we do. Trying it without the preprocessor block now.

Reply:
Superfluous preprocessor block removed. Seems to work fine.